### PR TITLE
installation: fix invenio_base imports

### PR DESCRIPTION
**Pull request by SamiHiltunen**
* FIX Adds missing `invenio_base` dependency and amends past
  upgrade recipes following its separation into standalone package.

Signed-off-by: Sami Hiltunen <sami.mikael.hiltunen@cern.ch>

### CODE REVIEW

File: NEWS [hunk 1]
```diff
@@ -1347,7 +1347,7 @@ Bug fixes:
 
 + docs:
 
-  - Adds missing `invenio.base` package to the `config.py` file for a
```
Comment (jirikuncar) [inline NEWS]: past NEWS should not be modified

File: docs/base.rst [hunk 2]
```diff
@@ -18,23 +18,23 @@
 Base
```
Comment (jirikuncar) [inline docs/base.rst]: @SamiHiltunen can you move this file to *Invenio-Base* package? Thanks

**Comment by jirikuncar** [comment]
@SamiHiltunen please rebase.

File: setup.py [hunk 3]
```diff
@@ -101,7 +101,7 @@ def run(self):
     "intbitset>=2.0",
     "invenio-access>=0.1.0",
     "invenio-accounts>=0.1.2",
-    "invenio-base>=0.2.0",
+    'invenio-base>=0.2.1',
```
Comment (jirikuncar) [inline setup.py]: please keep double quotes for consistency. thanks

**Comment by SamiHiltunen** [comment]
@jirikuncar Rebased now, updated the copyrights and changed the ' to " setup.py

**Pull request merged**